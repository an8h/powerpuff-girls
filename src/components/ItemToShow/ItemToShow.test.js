import React from "react";
import renderer from "react-test-renderer";
import ItemToshow from "./index";

const props = {
  item: {
    name: "This is a test name",
    summary: "This is a test summary",
    image: {
      medium: "",
      original: "",
    },
    rating: "5,6",
  },
};

describe("ItemToshow Component", () => {
  let component;

  beforeEach(() => {
    component = renderer.create(<ItemToshow item={props.item} />);
  });

  it("should render correctly", () => {
    expect(component.toJSON()).toMatchSnapshot();
  });
});
