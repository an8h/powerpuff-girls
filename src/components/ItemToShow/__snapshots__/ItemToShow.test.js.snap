// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ItemToshow Component should render correctly 1`] = `
<div>
  <div
    className="container"
  >
    <img
      alt="This is a test name"
      className="image"
      onError={[Function]}
      src=""
    />
    <div
      className="text"
    >
      <h2
        className="header"
      >
        This is a test name
      </h2>
      <div
        dangerouslySetInnerHTML={
          Object {
            "__html": "This is a test summary",
          }
        }
      />
    </div>
  </div>
</div>
`;
